### ISSUE TITLE
Login button truncated coming from wordlist

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alpheios-core#560]
the Login/Logout button is truncated in latin library when navigating from wordlist to user account (I did clear cache before installation)
see pics
![Screen Shot 2020-11-17 at 5 04 30 PM](https://user-images.githubusercontent.com/41396793/99428403-e4cb3f80-2906-11eb-8927-2233ba5fc947.png)
![Screen Shot 2020-11-17 at 5 43 41 PM](https://user-images.githubusercontent.com/41396793/99428425-eac12080-2906-11eb-9e97-8aa5ddf25b6c.png)

**Comment by monzug** [comment]
Actually, I am experiencing this 'issue' also in Safari App Extension 3.3.2 build 20200821381
Alpheios Components 3.3.1-20200821376  browsing through Alpheios panels
e.g. lookup for nero, then from morphology pop-up click on Usage Examples, the panel will not open on top of the page but slightly down
![Screen Shot 2020-11-18 at 4 33 04 PM](https://user-images.githubusercontent.com/41396793/99551340-020e1580-29bc-11eb-8f45-93c3362242e7.png)

**Comment by monzug** [comment]
![Screen Shot 2020-11-18 at 4 33 04 PM](https://user-images.githubusercontent.com/41396793/99551412-1ce08a00-29bc-11eb-88e7-63c106c9757c.png)

**Comment by monzug** [comment]
![Screen Shot 2020-11-18 at 4 32 16 PM](https://user-images.githubusercontent.com/41396793/99551478-2e299680-29bc-11eb-812d-4f7164d44aa1.png)

**Comment by monzug** [comment]
![Screen Shot 2020-11-18 at 4 28 32 PM](https://user-images.githubusercontent.com/41396793/99551595-56b19080-29bc-11eb-93c8-28378b3f707e.png)

**Comment by monzug** [comment]
I realized that it is more visible in Safari but the slightly top truncation does occur also in Chrome.

**Comment by balmas** [comment]
I cannot reproduce this one.  Moving it out of the incremental.

**Comment by balmas** [comment]
@kirlat I was not able to reproduce but if you are able to , please make the fix in a branch off the incr-3.3.x branch

**Comment by kirlat** [comment]
I cannot reproduce it either, no matter what I try. An interesting thing is that on all @monzug screenshots there is no scrollbars on any of the images, even if the content does not fit the panel. Here is a screenshot with the scrollbar from my tests for comparison:
![image](https://user-images.githubusercontent.com/18631055/106025384-1f5d0f80-60e2-11eb-8ba1-2e996a2ff11e.png)
Please compare it with the screenshot from https://github.com/alpheios-project/alpheios-core/issues/560#issuecomment-729761725 and the difference between them is the presence/absence of the scrollbar.

Maybe something (some CSS rules on the page, something else) prevents the panel content from scrolling and that may cause its content to shift upwards in order to fill all space available, even if it'll mean it will move behind the panel's header. But that's just my guess and it's really hard to tell without being able to reproduce it. It might also depend on the screen resolution and other factors that are individual for a particular machine and environment.

**Comment by monzug** [comment]
this is really a super minor thing. anyhow, my environment is Safari 14.0.2 MacOs Big Sur. the scroll bars are hidden until I activate them by scrolling.  I was able to repro in Safari with a wordlist longer than visible page, then scroll down a little, next go to login or grammar or any other options and you'll see only a portion of the top or nothing at all (depending how much I scrolled down on wordlist). Hope this can help. not sure if this was the original problem though.

**Comment by kirlat** [comment]
> my environment is Safari 14.0.2 MacOs Big Sur

That might be thing that makes a difference. I was testing it in Safari 13 on macOS Catalina. If I'll upgrade to Big Sur and Safari 14 that goes with it there will be no way back for me other than to reinstall the OS. But in order to provide compatibility we still has to test it with Safari 13 so it's probably better to stay on the older version for a while. I can only use BrowserStack to get screenshots but it's limited and does not allow to run the inspector and check the CSS rules.

**Comment by monzug** [comment]
Unfortunately with BrowserStack we have limited options for testing.

**Comment by irina060981** [comment]
>That might be thing that makes a difference. I was testing it in Safari 13 on macOS Catalina. If I'll upgrade to Big Sur and Safari 14 that goes with it there will be no way back for me other than to reinstall the OS. But in order to provide compatibility we still has to test it with Safari 13 so it's probably better to stay on the older version for a while. I can only use BrowserStack to get screenshots but it's limited and does not allow to run the inspector and check the CSS rules.

@kirlat , actually - you could open Inspect Item and see styles:

you could connect manually
click Live in the menu -> you would see this dashboard

![image](https://user-images.githubusercontent.com/12377640/106115374-72939880-619c-11eb-9f62-f962e5d663c8.png)

Choose the appropriate environment - I chose Mac OS Catalina Safari 13.1

It will open it via remote access

there you could click right mouse button click and select Inspect element

![image](https://user-images.githubusercontent.com/12377640/106115569-a53d9100-619c-11eb-9c37-25a54ce7b454.png)

![image](https://user-images.githubusercontent.com/12377640/106115680-c1413280-619c-11eb-9848-ba65e21604ed.png)

Hope, it will help

**Comment by monzug** [comment]
Thanks Irina. I used my daughter's old computer (Catalina, safari 14.0,
13.3 inch, 1440x900) and was not able to reproduce it. Mine is running Big
Sur, Safari 14.0.2, 13.3-inch (1440 × 900)

**Comment by kirlat** [comment]
> actually - you could open Inspect Item and see styles:

Thank you so much @irina! I did not realize that we can inspect styles in BrowserStack. Now I see it, will try to reproduce the bug. I see that Safari 14 hides the scrollbars by default (but that does not seem to cause any problems for the layout).

**Comment by monzug** [comment]
@kirlat, yes as I said above in Safari 14.0 and up, the scroll bars are hidden until I activate them by scrolling.

**Comment by kirlat** [comment]
Sorry, I can't reproduce it. Tried everything I can in BrowserStack, but this issue does not come up, no matter what I do. The limitation of the BrowserStack is that I can use it only on pages with the embedded library (because there is no way to test a webextension in BrowserStack) and that's the very limited amount of test cases. The issue may be caused by formatting quirks of other pages but they're out of reach for my testing.

@balmas, it's probably not to wise to spend to much time on this now. Maybe we can postpone?

**Comment by balmas** [comment]
Agree. removing from the project. thanks.
